### DISCUSSION TITLE
Esc key to close Slash commands window

### DISCUSSION THREAD

**Discussion by protie** [arnoudkooi/SN-Utils#146] [Ideas]
**As I user I would like to be able to close Slash commands window with a single key to speed up my work.**

**Current behavior on Chrome/Windows:**
If I accidentally press Esc key the focus moves to browser and it is impossible to get it back to Slash commands window via keyboard.
If I delete all characters from the Slash command window the window will eventually disappear but it takes too long to delete a long command.
Little red circle in the tope left corner closes the window too but it is tiny and can be confusing for Windows users.

**Why?**
- it is a common UI pattern - for example VS code, Sublime Text, Spotlight
- sometimes I try to do something via slash commands but than I realize that it cannot be done or that there is an easier way to achieve it and I want to be able to quickly dismiss the window

**Comment by arnoudkooi** [comment]
This is the expected behaviour. Hitting esc should close the window...

**Reply by protie** [reply]
You are right it works just fine. I just realized that it does not work on my machine because I also use browser extension called Vimium.
